services/sts/src/main/java/software/amazon/awssdk/services/sts/auth/StsCredentialsProvider.java

```diff
@@ -127,6 +150,30 @@ public B asyncCredentialUpdateEnabled(Boolean asyncCredentialUpdateEnabled) {
             return (B) this;
         }
 
+        /**
+         * Configure the amount of time, relative to STS token expiration, that a new STS token should be fetched synchronously.
+         *
+         * <p>By default, this is 1 minute.</p>
+         */
+        @SuppressWarnings("unchecked")
+        public B staleTime(Duration staleTime) {
+            this.staleTime = staleTime;
+            return (B) this;
+        }
+
+        /**
+         * Configure the amount of time, relative to STS token expiration, that a new STS token should be fetched asynchronously.
+         * The value only applies if asyncCredentialUpdateEnabled is enabled.
```

Comment:
Can we change this to: Configure the amount of time, relative to STS token expiration, that the cached credentials are considered close to stale and should be updated. See {@link #asyncCredentialUpdateEnabled}.

Reply:
Done